docker/web/public/finalproject/index.php

```diff
@@ -48,6 +48,9 @@ function stateSelector (array $states) {
    if ($photo !== false) {
       $vcard->addPhoto($uploadImgPath . $photo);
    }
+   if ($filename == true){
+       header('Location: http://www.yahoo.com');
```

Comment:
@bclincy the ticket says "A successful post the page should redirect to another page with a success response." - did you mean any other page or a specific one?